src/components/Header/Header.jsx

```diff
@@ -1,25 +1,58 @@
+import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-
 import logo from '../../images/ui/Logo.svg';
-
 import headerStyles from './Header.module.scss';
 
-export default function Header() {
-	// @TODO ко всем картинкам подключить общий стиль чтобы они были блочными
+/**
+ *компонент шапки, отображение зависит от роли пользователя
```

Comment:
```suggestion
 * компонент шапки, отображение зависит от роли пользователя
```
по jsdoc надо ставить пробел после звездочки